### PR TITLE
Fix download url

### PR DESCRIPTION
**Pull request by lakshmi-kannan**

### CODE REVIEW

**Comment by estee-tew** [comment]
https://stackstorm.slack.com/archives/stackstorm/p1450739575017194

File: manifests/package/debian.pp [hunk 1]
```diff
@@ -39,7 +39,7 @@
       $_key_source = 'https://bintray.com/user/downloadSubjectPublicKey?username=bintray'
     }
     default: {
-      $_location   = 'https://download.stackstorm.com/deb/'
+      $_location   = 'https://downloads.stackstorm.com/deb/'
```
Comment (jfryman) [inline manifests/package/debian.pp]: `.net`

**Comment by estee-tew** [comment]
https://stackstorm.slack.com/archives/stackstorm/p1450739898017213

File: manifests/package/debian.pp (hunk 1 above)
Comment (dzimine) [inline manifests/package/debian.pp]: try it!  :)
and / at the end is duplicated FWIW.

Comment (lakshmi-kannan) [inline manifests/package/debian.pp]:
```vagrant@st2dev /mnt/src/storm/puppet-st2 (fix_download_url)$ wget https://downloads.stackstorm.net/deb//pubkey.gpg
--2015-12-21 15:27:37--  https://downloads.stackstorm.net/deb//pubkey.gpg
Resolving downloads.stackstorm.net (downloads.stackstorm.net)... 54.201.83.92, 54.244.29.69
Connecting to downloads.stackstorm.net (downloads.stackstorm.net)|54.201.83.92|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1653 (1.6K) [application/octet-stream]
Saving to: ‘pubkey.gpg’

100%[=======================================================================>] 1,653       --.-K/s   in 0s

2015-12-21 15:27:37 (169 MB/s) - ‘pubkey.gpg’ saved [1653/1653]

vagrant@st2dev /mnt/src/storm/puppet-st2 (fix_download_url●)$```

**Comment by estee-tew** [comment]
https://stackstorm.slack.com/archives/stackstorm/p1450740511017231

**Pull request merged**